src/lib/phdi-building-blocks/phdi_building_blocks/utils.py

```diff
@@ -2,9 +2,30 @@
 
 
 def find_patient_resources(bundle: dict) -> List[dict]:
```

Comment:
I think this function should be generalized such that it's `find_resource_by_type(bundle: dict, resource:str) -> List[dict]`.